{"uid":"dcb9da2ec2ac05e","name":"test_legacy_endpoint_with_multiple_related_endpoints","fullName":"tests.integration.relations.test_relations#test_legacy_endpoint_with_multiple_related_endpoints","historyId":"9524a147699d384c1be0cb3464cff7ae","time":{"start":1728437833033,"stop":1728437924686,"duration":91653},"status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/relations/test_relations.py\", line 103, in test_legacy_endpoint_with_multiple_related_endpoints\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(10)):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 398, in <lambda>\n    self._add_action_func(lambda rs: rs.outcome.result())\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/relations/test_relations.py\", line 105, in test_legacy_endpoint_with_multiple_related_endpoints\n    with pytest.raises(psycopg2.OperationalError):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 1011, in __exit__\n    fail(self.message)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 178, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'psycopg2.OperationalError'>","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"ops_test","time":{"start":1728437600510,"stop":1728437600726,"duration":216},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728437600510,"stop":1728437600510,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1728437600510,"stop":1728437600510,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728437833033,"stop":1728437833033,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/relations/test_relations.py\", line 103, in test_legacy_endpoint_with_multiple_related_endpoints\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(10)):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 398, in <lambda>\n    self._add_action_func(lambda rs: rs.outcome.result())\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/relations/test_relations.py\", line 105, in test_legacy_endpoint_with_multiple_related_endpoints\n    with pytest.raises(psycopg2.OperationalError):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 1011, in __exit__\n    fail(self.message)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 178, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'psycopg2.OperationalError'>","steps":[],"attachments":[{"uid":"326eb83e75838518","name":"log","source":"326eb83e75838518.txt","type":"text/plain","size":1117}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1728437924745,"stop":1728437924745,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1728438051078,"stop":1728438051431,"duration":353},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728438051432,"stop":1728438051432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728438051432,"stop":1728438051432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728438051432,"stop":1728438051432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.relations"},{"name":"suite","value":"test_relations"},{"name":"host","value":"runner"},{"name":"thread","value":"17613-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.relations.test_relations"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d08f299a200cc643","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":0,"skipped":2,"passed":21,"unknown":3,"total":29},"items":[{"uid":"5ca5e54e2e9d3be9","reportUrl":"../104//#testresult/5ca5e54e2e9d3be9","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728351198602,"stop":1728351198602,"duration":0}},{"uid":"f056b0977b5aac96","reportUrl":"../103//#testresult/f056b0977b5aac96","status":"passed","time":{"start":1728265032380,"stop":1728265148597,"duration":116217}},{"uid":"efe39b9be72aa94b","reportUrl":"../102//#testresult/efe39b9be72aa94b","status":"passed","time":{"start":1728178553982,"stop":1728178675637,"duration":121655}},{"uid":"d369247e426e911","reportUrl":"../101//#testresult/d369247e426e911","status":"passed","time":{"start":1728092090249,"stop":1728092204827,"duration":114578}},{"uid":"2af7500a7ce7d5bd","reportUrl":"../100//#testresult/2af7500a7ce7d5bd","status":"passed","time":{"start":1728005760643,"stop":1728005874926,"duration":114283}},{"uid":"636ad9580ec7e91f","reportUrl":"../99//#testresult/636ad9580ec7e91f","status":"passed","time":{"start":1727919354041,"stop":1727919470614,"duration":116573}},{"uid":"aff282f671332883","reportUrl":"../98//#testresult/aff282f671332883","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","time":{"start":1727832947323,"stop":1727833037283,"duration":89960}},{"uid":"d6dc3d7464f52b8f","reportUrl":"../97//#testresult/d6dc3d7464f52b8f","status":"passed","time":{"start":1727746568928,"stop":1727746683526,"duration":114598}},{"uid":"5c8a8bda6ccf02","reportUrl":"../96//#testresult/5c8a8bda6ccf02","status":"passed","time":{"start":1727660148163,"stop":1727660266592,"duration":118429}},{"uid":"a81723619d1546fc","reportUrl":"../95//#testresult/a81723619d1546fc","status":"passed","time":{"start":1727573751470,"stop":1727573866354,"duration":114884}},{"uid":"a9a461a6e4fe0aaf","reportUrl":"../94//#testresult/a9a461a6e4fe0aaf","status":"passed","time":{"start":1727487296657,"stop":1727487411083,"duration":114426}},{"uid":"ca0755a091688b01","reportUrl":"../93//#testresult/ca0755a091688b01","status":"passed","time":{"start":1727400957677,"stop":1727401069038,"duration":111361}},{"uid":"f83676223cf3e6c1","reportUrl":"../92//#testresult/f83676223cf3e6c1","status":"passed","time":{"start":1727314700776,"stop":1727314817381,"duration":116605}},{"uid":"e70eb404f7019183","reportUrl":"../90//#testresult/e70eb404f7019183","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1722216989813,"stop":1722216989813,"duration":0}},{"uid":"efc5b077cd218667","reportUrl":"../89//#testresult/efc5b077cd218667","status":"passed","time":{"start":1722130718360,"stop":1722130843952,"duration":125592}},{"uid":"3ba9a66cf12a259c","reportUrl":"../88//#testresult/3ba9a66cf12a259c","status":"passed","time":{"start":1722044412079,"stop":1722044531207,"duration":119128}},{"uid":"1907b9de10e42495","reportUrl":"../87//#testresult/1907b9de10e42495","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","time":{"start":1721957856413,"stop":1721957954692,"duration":98279}},{"uid":"f3bf248aa8a8599b","reportUrl":"../86//#testresult/f3bf248aa8a8599b","status":"passed","time":{"start":1721871451169,"stop":1721871575062,"duration":123893}},{"uid":"5ba90c4f12cb0aa8","reportUrl":"../85//#testresult/5ba90c4f12cb0aa8","status":"passed","time":{"start":1721785669285,"stop":1721785787321,"duration":118036}},{"uid":"932737d8dd05e882","reportUrl":"../84//#testresult/932737d8dd05e882","status":"passed","time":{"start":1721699367084,"stop":1721699488397,"duration":121313}}]},"tags":["asyncio","group(1)"]},"source":"dcb9da2ec2ac05e.json","parameterValues":[]}